libs/deso-protocol/src/lib/identity/Identity.ts

```diff
@@ -23,12 +23,15 @@ import { BASE_IDENTITY_URI } from '../state/BaseUri';
 export interface IdentityConfig {
   node: Node;
   uri?: string;
+  testnet?: boolean;
```

Comment:
Change this to an enum and have the values be "mainnet" and "testnet". We may have more networks down the road and we grow